noir-projects/Earthfile

```diff
@@ -41,10 +41,25 @@ build:
   SAVE ARTIFACT noir-protocol-circuits
 
 test:
-  FROM +build
-  RUN cd noir-protocol-circuits && nargo test --silence-warnings
-  RUN cd aztec-nr && nargo test --silence-warnings
-  RUN cd noir-contracts && nargo test --silence-warnings
+  FROM ../yarn-project/+txe
```

Comment:
This feels problematic, but we need TXE to be compiled and listening to run `noir-contracts` and `aztec-nr` tests now. Would love some input CC @ludamad